### PR TITLE
jenkins v25

### PR DESCRIPTION
**Pull request by cprovencher**
@falseperfection: "Pass through the Jenkins Content Security Policy from the Universe configuration to the underlying package. See also: mesosphere/dcos-jenkins-service#163"

### CODE REVIEW

File: repo/packages/J/jenkins/25/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "jenkins-341-2892": "mesosphere/jenkins:3.4.0-2.89.2"
```
Comment (falseperfection) [inline repo/packages/J/jenkins/25/resource.json]: Number mismatch. jenkins-34`1`-2892 vs mesosphere/jenkins:3.4.`0`-2.89.2

package.json has "version": "3.4.`1`-2.89.2",

File: repo/packages/J/jenkins/25/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      {"port": 0, "protocol": "tcp", "name": "jenkins"}
+  ],
+  "container": {
+       "type": "DOCKER",
+       "docker": {
+       {{#advanced.docker-image}}
+           "image": "{{advanced.docker-image}}",
+       {{/advanced.docker-image}}
+       {{^advanced.docker-image}}
+           "image": "{{resource.assets.container.docker.jenkins-340-2892}}",
```
Comment (falseperfection) [inline repo/packages/J/jenkins/25/marathon.json.mustache:44]: Version check: jenkins-34`0`-2892

File: repo/packages/J/jenkins/25/resource.json (hunk 1 above)
Comment (cprovencher) [reply repo/packages/J/jenkins/25/resource.json]: oups. corrected

**Review by pcrews — APPROVED** [review]

**Review by vishnu2kmohan — APPROVED** [review]
I'm not the most knowledgeable about the Jenkins framework but it LGTM.
Feel free to merge if you feel it's ready or wait for one more Ship It, if you must :)

**Pull request closed without merging**